M/Mineos/build_tarballs.jl

```diff
@@ -3,19 +3,37 @@
 using BinaryBuilder
 
 name = "Mineos"
-version = v"1.0"
+version = v"1.0.2" 
 
 # Collection of sources required to build Mineos
 sources = [
-    "https://github.com/anowacki/mineos.git" =>
-    "e2558b486d7656ef112608a8776643da66dc87cf",
+    GitSource("https://github.com/geodynamics/mineos.git", "3dd7c7433766d630b929d8254d03e705808ff8a3"),
+    DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd ${WORKSPACE}/srcdir/mineos
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-doc
+
+# Drop docs build target
+atomic_patch -p1 ../patches/drop_docs.diff
```

Comment:
Is there a reason to apply a patch rather than use the official `--disable-doc` flag to `configure`?  I'm sure there may be good reasons.